lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -7,39 +7,78 @@
 
 /* eslint-env jest */
 
-require('../test-utils.js').makeMocksForGatherRunner();
-
-const Gatherer = require('../../gather/gatherers/gatherer.js');
-const GatherRunner_ = require('../../gather/gather-runner.js');
-const assert = require('assert').strict;
-const Config = require('../../config/config.js');
-const unresolvedPerfLog = require('./../fixtures/unresolved-perflog.json');
-const LHError = require('../../lib/lh-error.js');
-const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
-const Driver = require('../../gather/driver.js');
-const Connection = require('../../gather/connections/connection.js');
-const {createMockSendCommandFn, createMockOnceFn} = require('./mock-commands.js');
-const {
+import {jest} from '@jest/globals';
+import Gatherer from '../../gather/gatherers/gatherer.js';
+// import GathererRunner_ from '../../gather/gather-runner.js';
+import {strict as assert} from 'assert';
+// import Config from '../../config/config.js';
+import unresolvedPerfLog from './../fixtures/unresolved-perflog.json';
+import LHError from '../../lib/lh-error.js';
+import networkRecordsToDevtoolsLog from '../network-records-to-devtools-log.js';
+// import Driver from '../../gather/driver.js';
```

Comment:
rm commented out imports

Reply:
I purposefully left them as I wish us to be able to easily undo the import workarounds, if that becomes an option.